bin/update-analytics-vendors.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * External dependencies
+ */
+const { once } = require( 'events' );
+const fs = require( 'fs' );
+const readline = require( 'readline' );
+const stream = require( 'stream' );
+const axios = require( 'axios' );
+
+/**
+ * File path of the analytics vendors list.
+ */
+const ANALYTICS_VENDORS_FILE = 'data/analytics-vendors-list.json';
```

Comment:
Let's put this rather in `includes` alongside the `includes/ecosystem-data` directory.

Reply:
Resolved, added it to `includes` as `includes/data/analytics-vendors-list.php`